### ISSUE TITLE
Connection lost . Reconnecting when open a Mic in a camera over frigate card over HTTPS

### ISSUE THREAD

**Issue by kiloptero** [home-assistant/iOS#2420]
**iOS device model, version and app version**

Iphone 12pro 17.0.3
companion app 2023.7 446

Model Name: 
Software Version: 
App version: 2023.7 446

**Home Assistant Core Version**

Home Assistant 2023.10.1
Supervisor 2023.10.0
Operating System 10.5
Frontend 20231005.0 - latest
**Describe the bug**

**To Reproduce**
The connection over https works fine (using nginx) until the mic over a frigate card or something else.. Then after a few sec lost connection. only happened when open some kind of bidirectional comunication. T

sometimes the app crash and freeze and need to close and open again.

frigate is the example because I don't fave any other card can open a channel to the other via
**Expected behavior**

**Screenshots**

**Additional context**

**Comment by senguendk** [comment]
I can confirm exactly this. The companion app seems to "lose connection" or crash. There is nothing worth reporting in the logs at all. 

I am using a frigate card that loads camera stream using webrtc. Enabling the microphone for two-way audio crashes (or disconnects?) the app immediately.

**Comment by lorenzomartini1** [comment]
I've got the same here

**Comment by kiloptero** [comment]
same

Andres Roepke Del Solar.
Ingeniero Civil Industrial
Ingeniero Electronico
Mobile 88196932

**Comment by Martinvdm** [comment]
same issue, when using microphone in the stream, it takes 30 sec and Home assistant companion app is losing connection with home assistant

**Comment by bgoncal** [comment]
Did you provide the app access to microphone? Also, how does it behave in iOS safari?

**Comment by Martinvdm** [comment]
Yes I do allow access to the mic. You also do see the mic on top of the iPhone when accessed. In safari the same behavior is there. Also loosing connection.

**Comment by bgoncal** [comment]
Since it also happens in Safari, this is not an app specific issue, the app only displays a webview, which probably then makes more sense to move this issue to frontend repo.

**Comment by Martinvdm** [comment]
Thanks. Probably that’s best idee. Perhaps iOS is using underlaying safari was my thought.
